frontend/src/employee-frontend/components/unit/unit-reservations/attendance-elements.tsx

```diff
@@ -46,7 +46,10 @@ const CustomTh = styled(Th)<{ shrink?: boolean }>`
     `}
 `
 
+// Use min- and max-width to ensure that columns in two tables in the same layout are aligned
 const DateTh = styled(CustomTh)<{ faded: boolean }>`
+  min-width: 150px;
+  max-width: 150px;
```

Comment:
Is there a difference in `min-width` + `max-width` as opposed to just `width`? 🤔

Reply:
`width` alone will not force the width in this case (I think it's due to how `table`s do their layout, but I might be slightly wrong).